charts/refinery/values.yaml

```diff
@@ -206,6 +236,15 @@ config:
     # the first IPV6 unicast address found.
     UseIPV6Identifier: false
 
+    # Strategy controls the way that traces are assigned to refinery nodes.
+    # The "legacy" strategy uses a simple algorithm that unfortunately causes
+    # 1/2 of the in-flight traces to be assigned to a different node whenever the
+    # number of nodes changes.
+    # The legacy strategy is deprecated and is intended to be removed in a future release.
+    # The "hash" strategy is strongly recommended, as only 1/N traces (where N is the
+    # number of nodes) are disrupted when the node count changes.
+    Strategy: "legacy"
```

Comment:
Given that you're adding this stanza for the first time, might it make sense to just set it to "hash"?

Reply:
I think it would be considered a breaking change for the same reason we didn't default refinery to hash. If we set the default value to hash in the chart and someone takes an upgrade without setting it back to legacy then they just started using hash without opting in.